app/views/documents/_history_notice.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="history-status-block">
+  <h2>This <%= type.downcase %> is from a previous government</h2>
+  <% if government_name %>
```

Comment:
Is this conditional necessary? As above, if the document is historic, it must have a government, which must also have a name, which means this variable will always be set?